src/Impostor.Server/Net/State/Game.Outgoing.cs

```diff
@@ -100,5 +120,27 @@ private void WriteWaitForHostMessage(IMessageWriter message, bool clear, IClient
         {
             Message12WaitForHostS2C.Serialize(message, clear, Code, player.Client.Id);
         }
+
+        private ValueTask SendObjectSpawn(InnerNetObject obj, int? targetClientId = null)
```

Comment:
Game.Outgoing.cs contains 3 groups of methods:
- Senders (SendToAllAsync, SendToAllExceptAsync, SendToAsync)
- Start/Finish based API's (StartRpc, FinishRpc and the new GameData variants)
- Writers where you need to get your own writer and pick your own sender (Write*Message)

SendObjectSpawn is slightly different than the existing groups as it serializes a message like a writer, but relies on a Start/Finish API instead of letting a user pick their own.

As it doesn't modify Impostor's internal state, I don't think calling it SpawnAsync would be accurate.